docs/UserGuide.md

```diff
@@ -46,17 +51,66 @@ A GUI similar to the below should appear in a few seconds. Note how the app cont
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Introduction
+
+Before diving deeper into DDD's features, it will be helpful to explain a bit more about what DDD is trying to achieve. In short, DDD offers a comprehensive way of managing your contacts. As an extension, contacts can be also be tied to events.
```

Comment:
> Before diving deeper into DDD's features, it will be helpful to explain a bit more about what DDD is trying to achieve.

Phrasing sounds a bit like a presentation, I think we can make it more like "talking" to a user
Maybe: "Before diving deeper into DDD's features, let's run you through what DDD is trying to achieve."